lib/Target/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_subdirectory(AMDGCN)
```

Comment:
Since this PR already takes care of AMDGCN generation, please remove this subdirectory and references to it.

This was added in a previous commit.